packages/oauth/src/state-stores/clear-state-store.ts

```diff
@@ -1,23 +1,54 @@
 import { sign, verify } from 'jsonwebtoken';
 import { InstallURLOptions } from '../install-url-options';
 import { StateStore, StateObj } from './interface';
+import { InvalidStateError } from '../errors';
 
 // default implementation of StateStore
 export default class ClearStateStore implements StateStore {
   private stateSecret: string;
 
-  public constructor(stateSecret: string) {
+  private stateExpirationSeconds: number;
+
+  public constructor(
+    stateSecret: string,
+    stateExpirationSeconds: number = 600,
+  ) {
     this.stateSecret = stateSecret;
+    this.stateExpirationSeconds = stateExpirationSeconds;
   }
 
-  public async generateStateParam(installOptions: InstallURLOptions, now: Date): Promise<string> {
-    return sign({ installOptions, now: now.toJSON() }, this.stateSecret);
+  public async generateStateParam(
+    installOptions: InstallURLOptions,
+    now: Date,
+  ): Promise<string> {
+    const source = {
+      installOptions,
+      now: now.toJSON(),
+      random: Math.floor(Math.random() * 1000000),
```

Comment:
The reason why I added this is that, while doing tests with StateStore, I found that the generated JWT values with the same installOptions at the same timing can be exactly the same value. To prevent this happening, I've added additional random value here. This one is optional and no breaking changes to existing custom `StateStore` implementations.